frontend/src/metabase/query_builder/components/notebook/Notebook.tsx

```diff
@@ -82,14 +83,14 @@ const Notebook = ({ className, updateQuestion, ...props }: NotebookProps) => {
   };
 
   return (
-    <NotebookRoot className={className}>
+    <div className={cx(N.notebookRoot)}>
       <NotebookSteps updateQuestion={handleUpdateQuestion} {...props} />
       {hasVisualizeButton && isRunnable && (
-        <Button medium primary style={{ minWidth: 220 }} onClick={visualize}>
+        <Button variant="filled" style={{ minWidth: 220 }} onClick={visualize}>
```

Comment:
This makes the "Visualize" button a bit thicker, fyi.
But, this is a part of our design system and it now follows the same style/pattern as all other Mantine-adapted buttons.

Before
![image](https://github.com/metabase/metabase/assets/31325167/b12e88b2-4812-4e91-b113-bdf46d2d9574)

Now
![image](https://github.com/metabase/metabase/assets/31325167/c97ddb40-fa6a-4610-922e-767b0ade3714)